<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WagonsForm</class>
 <widget class="QMainWindow" name="WagonsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер поездов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: #b1b1b1;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 10px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}
</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>981</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>1</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>881</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 10px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}
</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
